### PR TITLE
Adds a danger rule to remind us to include resolved files when updating

### PR DESCRIPTION
**Pull request by pedrovereza**
Saw #1668 and thought it was weird that @MoyaBot didn't drop by 😄

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -52,6 +52,8 @@ end
 podspec_updated = !git.modified_files.grep(/Moya.podspec/).empty?
 cartfile_updated = !git.modified_files.grep(/Cartfile/).empty?
 cartfile_resolved_updated = !git.modified_files.grep(/Cartfile.resolved/).empty?
+package_updated = !git.modified_files.grep(/Package.swift/).empty?
+package_resolved_updated = !git.modified_files.grep(/Package.resolved/).empty?
 
 if podspec_updated && (!cartfile_updated || !cartfile_resolved_updated)
```
Comment (sunshinejr) [inline Dangerfile]: Would you be up for adding SPM to the checks here as well? Seems like we check if Carthage was updated and CP not or CP was updated and Carthage not.

**Comment by sunshinejr** [comment]
Thanks @pedrovereza this is great! Let me know how do you feel about my comment, otherwise it's good to go 😉

File: Dangerfile (hunk 1 above)
Comment (SD10) [reply Dangerfile]: Yeah, I think this could be a beneficial addition. If we don't add it here, let's open an issue for it

**Comment by pedrovereza** [comment]
@sunshinejr @SD10 Agree, the addition makes sense! If any of you wanna go ahead and do it, feel free to push to this branch! I probably won't have time to do it until Thursday 😕

**Pull request merged**